mvp-spec/json-schemas/transmission-request.json

```diff
@@ -11,17 +11,25 @@
         "seed": {
             "$ref": "seed.json"
         },
+        "data": {
+            "$ref": "ids-and-preferences.json"
+        },
         "source": {
-            "description": "Signature based on input:\n```\ntransmission_request_receiver_domain        + '\\u2063' +\ntransmission_request.source.domain          + '\\u2063' + \ntransmission_request.source.timestamp       + '\\u2063' +\nseed.source.signature\n```",
+            "description": "Signature based on input:\n```\ntransmission_request_receiver_domain        + '\\u2063' +\nsource.domain          + '\\u2063' + \nsource.timestamp       + '\\u2063' +\nseed.source.signature\n```",
```

Comment:
Sorry, I'm confused. Where is the transmission request receiver? I don't see it in this schema.

Reply:
Thanks a lot for this comment. It pinpoints an inconsistency in the schema and the example. Fixed.